src/sentry/incidents/subscription_processor.py

```diff
@@ -633,21 +652,27 @@ def trigger_alert_threshold(
                     date_detected=self.last_update,
                     projects=[self.subscription.project],
                     activation=activation,
+                    subscription=self.subscription,
                 )
             # Now create (or update if it already exists) the incident trigger so that
             # we have a record of this trigger firing for this incident
-            incident_trigger = self.incident_triggers.get(trigger.id)
+            # NOTE: `incident_triggers` is derived from `self.active_incident`
+            incident_trigger = self.incident_trigger_map.get(trigger.id)
             if incident_trigger:
                 incident_trigger.status = TriggerStatus.ACTIVE.value
+                if not incident_trigger.subscription_id:
+                    # NOTE: until Incidents are per subscription, this will overwrite the subscription_id
+                    incident_trigger.subscription_id = self.subscription.id
```

Comment:
Why are we setting the subscription id on the trigger? I thought this was at the incident level?

Is this line covered by a test?

Reply:
oops - this was left over!

thanks for the catch 🙏